mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -23,12 +24,20 @@ def cross_entropy(pred,
         avg_factor (int, optional): Average factor that is used to average
             the loss. Defaults to None.
         class_weight (list[float], optional): The weight for each class.
+        ignore_index (int | None): The label index to be ignored.
```

Comment:
`ignore_index` may be confusing, no keyword in the name to indicate it  is only related  to `label`

Reply:
Is it right the type `int? Does it only support ignore one class?

Reply:
ignore_index is a key of F.cross_entropy_loss(). It only supports ignore one class. Usually, this number does not belong to the classes of a model.